WordPressKitTests/DomainsServiceRemoteRESTTests.swift

```diff
@@ -238,4 +243,37 @@ class DomainsServiceRemoteRESTTests: RemoteTestCase, RESTTestable {
 
         waitForExpectations(timeout: timeout, handler: nil)
     }
+
+    // MARK: - Get All Domains Tests
+
+    func testGetAllDomainsSucceeds() {
+        let expect = expectation(description: "Get All Domains Succeeds")
+
+        stub { request -> Bool in
+            let url = request.url
+            let matched = url?.path == self.allDomainsEndpoint && url?.query == "no_wpcom=true&resolve_status=true&locale=en"
+            XCTAssertTrue(matched)
+            return matched
+        } response: { request in
+            let path = OHPathForFile(self.allDomainsMockFilename, type(of: self)) ?? ""
```

Comment:
I think it would be okay to crash here, as if the expected file is not there, it's a developer error.

```suggestion
            let path = OHPathForFile(self.allDomainsMockFilename, type(of: self))!
```